dnfdaemon-client/commands/repoquery/repoquery.cpp

```diff
@@ -151,12 +160,13 @@ void CmdRepoquery::run(Context & ctx) {
         PackageDbus package(raw_package);
         if (info_option->get_value()) {
             // TODO(mblaha) use smartcols for this output
-            std::cout << "Name: " << package.get_name() << std::endl;
-            std::cout << "Epoch: " << package.get_epoch() << std::endl;
-            std::cout << "Version: " << package.get_version() << std::endl;
-            std::cout << "Release: " << package.get_release() << std::endl;
-            std::cout << "Architecture: " << package.get_arch() << std::endl;
-            std::cout << "Repository: " << package.get_repo() << std::endl;
+            libdnf::cli::output::print_package_info_table(package);
+            // std::cout << "Name: " << package.get_name() << std::endl;
+            // std::cout << "Epoch: " << package.get_epoch() << std::endl;
+            // std::cout << "Version: " << package.get_version() << std::endl;
+            // std::cout << "Release: " << package.get_release() << std::endl;
+            // std::cout << "Architecture: " << package.get_arch() << std::endl;
+            // std::cout << "Repository: " << package.get_repo() << std::endl;
```

Comment:
If the code commented out is replaced by `cli::output::print_package_info_table` then remove it instead of commenting out.